pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -5,43 +5,88 @@ title: Upgrade Kommander
 menuWeight: 20
 excerpt: Steps to upgrade Kommander via CLI
 beta: false
-enterprise: true
 ---
  
-This section describes how to upgrade your Kommander Management cluster and all Platform Applications to their latest versions. To prevent compatibility issues, you must first upgrade Kommander on your Management Cluster before upgrading to DKP.
+This section describes how to upgrade your Kommander Management cluster and all Platform Applications to their latest versions in networked, air-gapped and on-prem environments. To prevent compatibility issues, you must first upgrade Kommander on your Management Cluster before upgrading to DKP.
 
 <p class="message--note"><strong>NOTE: </strong>It is important you upgrade Kommander BEFORE upgrading the Kubernetes version (or Konvoy version for Managed Konvoy clusters) in attached clusters, due to the previous versions' incompatibility with 1.22.</p>
 
 ## Prerequisites
 
-- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- [Download][download_binary] and install the latest DKP CLI binary on your computer.
 - Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
-- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre_provisioned].
+- Review the [Platform Application version updates][release_notes] that are part of this upgrade.  
+- For air-gapped environments only: [Load the Docker images into your Docker registry][load_images]
```

Comment:
one thing I'm wondering is should we link here as well https://dev-docs.d2iq.com/dkp/kommander/2.2/install/air-gapped/catalog/#load-the-docker-images-into-your-docker-registry

In 2.2, since catalog applications is an enterprise only feature, we had to move them to a separate install page 😞

Reply:
```suggestion
- For air-gapped environments with DKP Catalog Applications (multi-cluster): [Load the Docker images into your Docker registry][load_images_catalog]
- For air-gapped environments without DKP Catalog Applications: [Load the Docker images into your Docker registry][load_images]
```